src/main/java/uk/gov/dhsc/htbhf/productionstub/model/v2/DWPEligibilityRequestV2.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.dhsc.htbhf.productionstub.model.v2;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+import java.time.LocalDate;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+@Data
+@Builder
+@AllArgsConstructor(onConstructor_ = {@JsonCreator})
+public class DWPEligibilityRequestV2 {
```

Comment:
Couldn't you import the new DWP module to bring these classes in?

Reply:
Done.